docs/source/additional_features.rst

```diff
@@ -7,3 +7,78 @@ including the following:
 - inheritance between generic classes
 - compatibility and subtyping of generic types, including covariance of generic types
 - ``super()``
+
+
+.. _attrs_package:
+
+The attrs package
+*****************
+
+`attrs <https://www.attrs.org/en/stable>`_ is a package that lets you define
+classes without writing boilerplate code. Mypy can detect uses of the
+package and will generate the necessary method definitions for decorated
+classes using the type annotations it finds.
+Type annotations can be added as follows:
+
+.. code-block:: python
+
+    import attr
+    @attr.s
+    class A:
+        one: int = attr.ib()          # Variable annotation (Python 3.6+)
+        two = attr.ib()  # type: int  # Type comment
+        three = attr.ib(type=int)     # type= argument
+
+If you're using ``auto_attribs=True`` you must use variable annotations.
+
+.. code-block:: python
+
+    import attr
+    @attr.s(auto_attribs=True)
+    class A:
+        one: int
+        two: int = 7
+        three: int = attr.ib(8)
+
+The Typeshed has a couple of "white lie" annotations to make type checking
```

Comment:
I might have been unclear (sorry), but I was mostly talking about removing the preceding article. Typeshed is a proper noun and the name is referred to without a preceding article everywhere else.